src/OrbitDB.js

```diff
@@ -369,22 +394,22 @@ let databaseTypes = {
 
   // Save the database locally
   async _addManifestToCache (directory, dbAddress) {
-    const cache = await this._loadCache(directory, dbAddress)
-    await cache.set(path.join(dbAddress.toString(), '_manifest'), dbAddress.root)
+    // const cache = await this._loadCache(directory, dbAddress)
```

Comment:
`directory` can be removed from args

Reply:
sorry, removed from the args of which function call? `_addManifestToCache`?

Reply:
Oops sorry yea I meant from `_addManifestToCache`